index.html

```diff
@@ -61,9 +62,16 @@ <h1 class="display-3">Bikes for Refugees</h1>
 							<a class="btn btn-secondary btn-lrg" href="#">Volunteer</a>
 						</div>
 					</div>
+					<div class="container">
```

Comment:
Be careful using containers. They are not really meant to be nested like this. Use one around you're whole webpage, and then just work with `row`s and `col-...`s